send-transaction-service/src/send_transaction_service.rs

```diff
@@ -646,7 +679,7 @@ mod test {
             &root_bank,
             &tpu_address,
             &mut transactions,
-            &None,
+            leader_info_provider.clone(),
```

Comment:
I think the cloning is necessary with the switch  to `as_ref()`?

Reply:
I have changed the process_transactions function take a reference -- so the cloning is no longer necessary.